src/strings.cpp

```diff
@@ -806,18 +828,26 @@ static std::vector<const char *> _game_script_raw_strings;
 
 /**
  * Parse most format codes within a string and write the result to a buffer.
- * @param buff    The buffer to write the final string to.
+ * @param builder The string builder to write the final string to.
  * @param str_arg The original string with format codes.
  * @param args    Pointer to extra arguments used by various string codes.
- * @param last    Pointer to just past the end of the buff array.
  * @param dry_run True when the argt array is not yet initialized.
```

Comment:
lol; `argt`. Sorry, not your fault, but typo :P If there is nothing else, you can ignore this comment :)